t/lib/OpenQA/SeleniumTest.pm

```diff
@@ -187,7 +187,12 @@ sub wait_for_ajax {
     my $slept          = 0;
     my $msg            = $args{msg} ? (': ' . $args{msg}) : '';
 
-    while (!$_driver->execute_script('return window.jQuery && jQuery.active === 0')) {
+    if (!$_driver->execute_script('window.jQuery')) {
```

Comment:
We include jQuery on each and every page. Since Bootstrap and DataTables are still depending on it that will also not change in the next time. So I doubt that this shortcut will ever be taken.